app/models/ocelot/errors/GuidanceErrors.scala

```diff
@@ -37,6 +37,7 @@ case class PhraseNotFound(id: String, index: Int) extends FlowError
 case class LinkNotFound(id: String, index: Int) extends FlowError
 case class DuplicatePageUrl(id: String, url: String) extends FlowError
 case class MissingWelshText(id: String, index: String, english: String) extends FlowError
+case class VisualStanzasAfterQuestion(id: String) extends FlowError
```

Comment:
Could we make this generic to cover all input types - so Question or new InputStanza being developed?  Guess depends if value in identifying it as a question or input stanza

Reply:
You could include visual stanzas after input fields. Questions are a bit different as GDS makes the question text bethe H1 on the page and the visual stanzas before the question are wrapped in the question, so the question becomes the whole page and no visual stuff can follow.